returnBundle.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 The x9 Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package x9
+
+// ReturnBundle contains forward items (checks)
+type ReturnBundle struct {
+	// ID is a client defined string used as a reference to this record.
+	ID string `json:"id"`
+	// BundleHeader is a Bundle Header Record for returns
+	BundleHeader *BundleHeader `json:"bundleHeader,omitempty"`
+	// Items are Items: Check Detail Records, Check Detail Addendum Records, and Image Views
+	Returns []*ReturnDetail `json:"items,omitempty"`
+	// BundleControl is a Bundle Control Record
+	BundleControl *BundleControl `json:"bundleControl,omitempty"`
+}
+
+// NewReturnBundle takes a BundleHeader and returns a ReturnBundle
+func NewReturnBundle(bh *BundleHeader) *ReturnBundle {
+	rb := new(ReturnBundle)
+	rb.SetControl(NewBundleControl())
+	rb.SetHeader(bh)
+	return rb
+}
+
+// Validate performs X9  validations and format rule checks and returns an error if not Validated
+func (rb *ReturnBundle) Validate() error {
+	return nil
```

Comment:
This was the only func I saw left todo!

Reply:
That's correct, 

I also have to do Validate for Bundle, CashLetter, and File.  There will also be additional validations other than correct value, format and inclusion.  For example (# of AddendaRecords equaling ReturnDetail.AddendumCount).   Coming soon..to X9.